modules/table/src/main/java/org/apache/ignite/internal/table/distributed/disaster/DisasterRecoveryManager.java

```diff
@@ -78,7 +90,7 @@ public class DisasterRecoveryManager implements IgniteComponent {
 
     private static final int TIMEOUT = 30;
 
-    private static final int CATCH_UP_THRESHOLD = 10;
+    private static final int CATCH_UP_THRESHOLD = 100;
```

Comment:
Could you please add javadocs to `TIMEOUT` and `CATCH_UP_THRESHOLD` explaining what is means and in which units they are measured? Also, maybe it makes sense to include the unit in the name for the timeout constant, like `TIMEOUT_SEC`.